pandas/tests/indexing/multiindex/test_multiindex.py

```diff
@@ -188,28 +188,17 @@ def test_subtracting_two_series_with_unordered_index_and_all_nan_index(
         self, data_result, data_expected
     ):
         # GH 38439
+        # TODO: Refactor. This is impossible to understand
```

Comment:
Nit: Could you add your GH issue reference?

Reply:
Already opened a issue for this, do you think this is sufficient?